software/CONFIGURATION.md

```diff
@@ -53,3 +53,84 @@ shows the default configuration:
 ```
 
 - `volts_per_octave` must be one of `1.0` (Eurorack standard) or `1.2` (Buchla standard)
```

Comment:
Is this still experimental? It's also listed in the default configuration above, which isn't experimental

Reply:
Yes; it was moved out of the main config and into experimental a while ago. Looks like the docs were never updated to reflect that.